### PR TITLE
Setup HoH KinD cluster with service-ca and router 

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

**Comment by yanmxa** [comment]
https://github.com/stolostron/backlog/issues/23507

File: test/setup/common.sh [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+# deploy olm
+function addOperatorLifecycleManager() {
+  CTX_HUB="$1"
+  kubectl config use-context "$CTX_HUB"
+  kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+  kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
+  olmMessage=$(kubectl get csv -n olm --ignore-not-found | grep "packageserver" | awk '{print $5}' | grep "Succeeded") 
+  while [[ -z "$olmMessage" ]]; do
+    echo "Waiting for OLM to become available"
+    kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
```
Comment (clyang82) [inline test/setup/common.sh:284]: can you pin to the latest tag?

File: test/setup/hoh_setup.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# kubeconfig
+KUBECONFIG=${CONFIG_DIR}/kubeconfig
+sleep 1 &
+hover $! "KUBECONFIG=${KUBECONFIG}" "$PID"
+
+# init hub-of-hubs cluster
+initKinDCluster "$HUB_OF_HUB_NAME" >> "${LOG}" 2>&1 & 
+hover $! "HoH Create KinD Cluster $HUB_OF_HUB_NAME" "$PID"
+enableRouter "kind-$HUB_OF_HUB_NAME" >> "$LOG" 2>&1
+enableServiceCA "kind-$HUB_OF_HUB_NAME" "${CURRENT_DIR}/service-ca/" >> "$LOG" 2>&1
```
Comment (clyang82) [inline test/setup/hoh_setup.sh:31]: maybe you can use this [operator](https://github.com/openshift/service-ca-operator) instead of put the manifests here?

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: test/setup/hoh_setup.sh (hunk 2 above)
Comment (yanmxa) [reply test/setup/hoh_setup.sh]: It seems the [service-ca-operator](https://github.com/openshift/service-ca-operator) doesn't exist in the [community](https://operatorhub.io/).

File: test/setup/common.sh (hunk 1 above)
Comment (yanmxa) [reply test/setup/common.sh]: sure

**Pull request merged**